components/inputs/input-text.js

```diff
@@ -678,6 +679,13 @@ class InputText extends InputInlineHelpMixin(PropertyRequiredMixin(FocusMixin(La
 			this._lastSlotWidth = 0;
 		}
 
+		requestAnimationFrame(() => {
+			this.dispatchEvent(new CustomEvent(
+				'd2l-input-text-layout-updated',
```

Comment:
Let me know thoughts on if `search` should be catching this and then dispatching its own layout change event. With the way this is done now, filter is assuming `input-text` is within `search`.

Reply:
Since we could probably consider this an "internal" event that only we should care about, I'd be tempted to do that yeah. That way we could turn off bubbling and make it not composed. But I don't feel strongly about that.